docs/2_4_common_co-simulation.adoc

```diff
@@ -179,24 +179,33 @@ Multiple event types and also <<outputClock>> ticks or interrupts can be support
 
 ==== Intermediate Variable Access [[intermediate-variable-access]]
 
-A Co-Simulation FMU can provide valid or temporary <<output>> variables at intermediate points between two consecutive communication points, or is able to accept new values for <<input>> variables at these intermediate points.
+A Co-Simulation FMU can provide values for its <<output>> variables at intermediate points between two consecutive communication points, and is able to accept new values for <<input>> variables at these intermediate points.
+This is typically required when the FMU uses a numerical solver to integrate the FMU's internal state between communication points in <<fmi3DoStep>>.
+This numerical solver assumes that the inputs are continuous in the integration interval, dictacted by <<fmi3DoStep>>.
+In FMI 2.0 Co-simulation, the intermediate inputs are provided by the use of extrapolations.
+The intermediate variable access functions allows the FMU to receive inputs, and provide outputs, directly to the master, in those intermediate time points.
 
-- Intermediate <<output>> variables are typically the result of an internal solver taking multiple integration steps between two consecutive communication points of a <<fmi3DoStep>>.
-It is possible for the FMU to inform the master whether the internal solver is in a temporary state or if it has just completed an internal integration step.
-The FMU exposes intermediate output variables for the master whenever they are available.
-If the internal integration step is completed, the master can forward intermediate outputs to other FMUs, where they can be used, for e.g., for extrapolation, interpolation, filtering or asynchronous co-simulation.
+Due to the way numerical solvers estimate and correct the approximation error, these intermediate <<output>> values may be tentative or may be final.
+It is possible for the FMU to inform the master whether the internal solver is in a tentative state, meaning that the output values computed from that state are also tentative, or if the internal solver has successfully completed the integration step, meaning that the FMU's internal state is final, and will never be change by the normal in the current execution of <<fmi3DoStep>>.
+If the internal integration step has successfully completed, the master can forward intermediate outputs to other FMUs, where they can be used, for e.g., for extrapolation, interpolation, filtering or asynchronous co-simulation.
+For tentative output values, the master must not forward these values to other FMUs.
```

Comment:
In yesterday's discussion I think we concluded not to explicitely forbid this in normative text, but explain that one cannot rely on these values.

Reply:
Ah ok! Sorry for that! I'll revert this asap!